src/java.base/share/classes/java/net/Inet4Address.java

```diff
@@ -180,6 +189,72 @@ public static Inet4Address ofLiteral(String ipv4AddressLiteral) {
         return parseAddressString(ipv4AddressLiteral, true);
     }
 
+    /**
+     * Creates an {@code Inet4Address} based on the provided {@linkplain
+     * Inet4Address##format textual representation} of an IPv4 address in
+     * POSIX {@code inet_addr} compatible form.
```

Comment:
```suggestion
     * Creates an {@code Inet4Address} based on the provided {@linkplain
     * Inet4Address##format-posix textual representation of an IPv4 address in
     * POSIX {@code inet_addr} compatible form}.
```

Alternatively you could just remove the link, since the format description is just below.